### DISCUSSION TITLE
报AttributeError

### DISCUSSION THREAD

**Discussion by AKsitaWa** [hect0x7/JMComic-Crawler-Python#315] [Q&A] [answered]
重装过python还是这样咋整
![image](https://github.com/user-attachments/assets/56a1210e-1ca2-4e97-8a28-d8b94659bd13)

**Comment by hect0x7** [comment, marked as answer]
不要把文件命名为jmcomic.py

**Reply by AKsitaWa** [reply]
OK OK 已解决
